src/components/Books.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './styles/Book.css';
+import Form from './Form';
+import BookList from './BookList';
+
+const Book = () => (
+  <section>
+    <BookList />
```

Comment:
- [OPTIONAL] I believe Teddy miss these out in his first review. This component needs to take in a data prop and renders the title and the author. Since your redux is not set up yet. You can use static array data loop through it and pass them to the `BookList`

See example:
```js
const testArray = [
      {
          title: "Test",
          author: "Test Author"
      },
];


// In your jsx now
{ testArray && testArray.map((data) => {
    <BookList props={data} />
}); }

// finally in your BookList destructure the data and render 
```